src/core/xtests/rhc_torture/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
 idlc_generate(RhcTypes RhcTypes.idl)
+include(${CMAKE_CURRENT_LIST_DIR}/../../../../cmake/Modules/CUnit.cmake)
```

Comment:
This probably shouldn't be here. I'm guessing this is required for setting the path property for the test, but pulling in the entire `CUnit.cmake` module seems like overkill.